core/src/main/java/io/opencensus/stats/View.java

```diff
@@ -75,7 +77,8 @@
   /**
    * Constructs a new {@link View}.
    *
-   * @param name the {@link Name} of view. Must be unique.
+   * @param name the {@link Name} of view. Must be unique. Suggested format for name:
+   *     {@code <web_host>/<path>}.
```

Comment:
I think the suggested format should be specified on Name.create, and Name.create should validate the String.